xclim/indicators/atmos/_precip.py

```diff
@@ -251,12 +256,27 @@ class HrPrecip(Hourly):
     description="{freq} total precipitation.",
     abstract="Total accumulated precipitation. If the average daily temperature is given, the phase parameter can be "
     "used to restrict the calculation to precipitation of only one phase (liquid or solid). Precipitation is "
-    "considered solid if the average daily temperature is below 0°C (and vice versa).",
+    "considered solid if the average daily temperature is below {thresh} (and vice versa).",
```

Comment:
```suggestion
    "considered solid if the average daily temperature is below 0°C (and vice versa).",
```

The convention we landed on for these templates is to only have fillable fields in the `description` and `long_name`

Reply:
We can definitely set the defaults at the indicator or use the defaults that are inherited from `indices`. Generally, the description and long_name should have information within it that more specifically details the way that this indicator was called (containing fillable fields, like `{thresh}`), while the abstract would be what the user expects to see when reading the docstring for the function (before having called it).

I think we should explain what is returned when calling it without having modified the call signature.

Reply:
I think I found the knot in my brain.

The indicators `xclim.indicators.atmos.precip_{accumulation|average}` are explicitly defined as indicators respecting both liquid **and** solid precipitation. If one will compute an indicator for either liquid **or** solid precipitation one can call `xclim.indicators.atmos.{liquid|solid}_precip_{accumulation|average}`.

Thus, we don't need this liquid-solid differentiation in `xclim.indicators.atmos.precip_{accumulation|average}` since we can find it in both `long_name` and `description` of `xclim.indicators.atmos.{liquid|solid}_precip_{accumulation|average}`.

Isn't it?

Reply:
Yes, precisely! `Indicators` make it simpler for users to return a specific variable with a different default call signature from the `indice`, which includes all CF-checks and without needing to change any settings.

There are a lot of layers to consider, but the way I envision it is:
- `indices.generic`: Building blocks for creating climate indexes (exposed but no checks for units, nothing specifically tied to a variable/unit).
- `indices.*`: Climate indexes, includes checks for proper units on both input and output datasets, lots of user control and granularity in settings (operators, thresholds, methods, etc.)
- `indicators.*`: Most user-friendly way of calling `indices.*` with all checks integrated. This is where very specific indexes can be defined with fixed default values and lots of finely-tuned metadata.